### PR TITLE
Explanation of how to use the API Methods with cURL

### PR DESCRIPTION
**Pull request by fraengii**
Maybe it is trivial to people who are familiar with using APIs but on the first view I did not understand what to do with the terms in the brackets in the explanations of the methods. From what I know about coding one gives variables or parameter to a function within the brackets but this doesn't work here.
So I thought it would help to explain this at first.

### CODE REVIEW

**Review by haruka98 — APPROVED** [review]

File: doc/api/http_api.md [hunk 1]
```diff
@@ -68,6 +68,7 @@ Portal submits content into new blog post
 The latest version is `1.2.14`
 
 The current version can be queried via /api.
+(curl "https://<YourEtherpad-Lite>/api")
```
Comment (rhansen) [inline doc/api/http_api.md:71]:
```suggestion
(`curl "https://<YourEtherpad-Lite>/api"`)
```

File: doc/api/http_api.md [hunk 2]
```diff
@@ -136,6 +137,10 @@ All functions will also be available through a node module accessible from other
 
 ## API Methods
 
+The terms in the curved brackets have to be appended at the end of the request with a '&'. The API method `deleteGroup(groupID)` used with cURL looks like:
```
Comment (rhansen) [inline doc/api/http_api.md:140]:
```suggestion
Arguments are passed via query parameters. For example, to call API method
`deleteGroup(groupID)` with cURL, you would run:
```

File: doc/api/http_api.md [hunk 3]
```diff
@@ -136,6 +137,10 @@ All functions will also be available through a node module accessible from other
...
+
+		curl "http:/pad.domain/api/<version>/deleteGroup?apikey=<API-key>&groupID=<YourDesiredGroupID>"
```
Comment (rhansen) [inline doc/api/http_api.md:142]:
````suggestion
```shell
curl "http://etherpad.example/api/<version>/deleteGroup?apikey=<API-key>&groupID=<YourDesiredGroupID>"
```
````

File: doc/api/http_api.md [hunk 4]
```diff
@@ -236,7 +241,7 @@ Sessions can be created between a group and an author. This allows an author to
 #### createSession(groupID, authorID, validUntil)
  * API >= 1
 
-creates a new session. validUntil is an unix timestamp in seconds
+creates a new session. validUntil is an [unix timestamp](https://www.unixtimestamp.com/) in seconds
```
Comment (rhansen) [inline doc/api/http_api.md:244]:
```suggestion
Creates a new session. validUntil is a [Unix
timestamp](https://www.unixtimestamp.com/) in seconds.
```

**Review by rhansen — COMMENTED** [review]
Thanks! I proposed some changes; feel free to reject them if you disagree.

**Pull request closed without merging**